### PR TITLE
new wallpapers

### PR DESCRIPTION
**Pull request by gitEugeneL**
_Your wallpaper set is amazing! Thank you so much._
I thought I'd add some of the wallpapers I have.

### CODE REVIEW

File: wallpapers/other/0377_mountain-dreamscape.jpg
Comment (amtoine) [inline wallpapers/other/0377_mountain-dreamscape.jpg:1]: i would put that one in `wallpapers/real/`

File: wallpapers/other/0378_spectacular-evening-sky-illustration.jpg
Comment (amtoine) [inline wallpapers/other/0378_spectacular-evening-sky-illustration.jpg:1]: i would put that one in `wallpapers/real/`

**Review by amtoine — CHANGES_REQUESTED** [review]
> *Your wallpaper set is amazing! Thank you so much.*
>
thanks for the kind words :blush: 
happy you like it :yum: 

> I thought I'd add some of the wallpapers I have.
>
ooh yeah for sure, that's a good idea, let's share wallpapers :partying_face: 

i'd just move two of the wallpapers and then let's land this :wink:

File: wallpapers/other/0377_mountain-dreamscape.jpg
Comment (gitEugeneL) [reply wallpapers/other/0377_mountain-dreamscape.jpg:1]: Yeah, of course I've already done that

File: wallpapers/other/0378_spectacular-evening-sky-illustration.jpg
Comment (gitEugeneL) [reply wallpapers/other/0378_spectacular-evening-sky-illustration.jpg:1]: Yeah, of course I've already done that

**Review by amtoine — APPROVED** [review]
thanks for moving the files @gitEugeneL :pray: 

let's land this then :yum:

**Pull request merged**